### PR TITLE
grammar updates revision 1

### PR DESCRIPTION
**Pull request by byronbytes**
I might change some of these later, but this is good so far (I think)

### CODE REVIEW

**Comment by byronbytes** [comment]
I was also wondering, is Discord Oxygen going to have an external UI? And how would it work? I'm really interested with this project and want to help support it to the best I can (I don't have money for the patreon though, but I would support it if I did)

**Review by hxr404 — APPROVED** [review]

**Comment by hxr404** [comment]
> I was also wondering, is Discord Oxygen going to have an external UI? And how would it work?

What do you mean with "external UI"?
Discord Oxygen will include a GUI module which integrates nicely into Discord's UI (e.g. adds stuff to the settings menu on behalf if the other modules)

In case you mean something like an extra UI outside of Discord, that's not planned, as dc oxygen itself is just a pure client mod / js library

The actual installer (with GUI) will be a separate project and it will be open-source (probably licensed under something like GPLv3+)

> I'm really interested with this project and want to help support it to the best I can (I don't have money for the patreon though, but I would support it if I did)

My long term goal here is to open-source as many parts of Discord Oxygen as possible, without compromising this project's security (some parts like the integrity check and core-components containing exploits for Discord will always remain closed-source)
So to answer your question:
at later stages in development there will be more possibilities to contribute, for example by writing installers or providing packaging and there will be a weblate (or something similar) where you can contribute translations (or fix grammar mistakes xD)

Btw thanks a lot for your help and willingness to contribute!

**Comment by byronbytes** [comment]
I think the PR forgot to get merged lol

**Comment by hxr404** [comment]
> I think the PR forgot to get merged lol

Yeah the pull request itself forgot to get merged, so it really wasn't my fault 😉
(Thanks btw for reminding me, I wasn't really sure whether you were done with this PR so I kinda forgot abt it)

**Comment by byronbytes** [comment]
Whoops, I think I broke something with the HTML code

File: README.md [hunk 1]
```diff
@@ -2,47 +2,47 @@
...
 <h2>About</h2>
-Discord Oxygen is a universal Client mod with many Features and every single Hack <strong>I ever created</strong>.
+Discord Oxygen is a universal client modification with many features and includes every single hack <strong>I've created</strong>.
 
 <h2>Important Information</h2>
 <p>
-  Many Features listed here are only client-side!<br>
-  This means that some things like a custom discriminator or animated profile picture/banner are <strong>only visibile to other users of Discord Oxygen!!!</strong><br>
+  Many features listed here are only client sided!<br>
+  This means that some things like a custom discriminator or animated profile picture/banner are <strong>only visible to other Discord Oxygen users!/strong><br>
```
[the remark this answers is not in the document]
Comment (byronbytes) [reply README.md:10]: Alright I'll fix that really quickly

[the remark this answers is not in the document]
Comment (hxr404) [reply README.md:10]: > Alright I'll fix that really quickly

no need to - I already fixed it myself after merging the PR
thanks for the offer tho

**Pull request merged**